.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,52 @@
+---
+name: Bug Report
+description: Report a bug to help us improve
+title: "[BUG] <BUG>"
+labels: ["bug"]
+body:
+  - type: input
+    attributes:
+      label: Plugin
+      description: The affected NixVim plugin(s)
+  - type: dropdown
+    attributes:
+      label: Nixpkgs Release
+      multiple: false
+      options:
+        - "unstable"
+        - "25.05"
+        - "24.11"
+        - "other"
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Home Manager Release
+      multiple: false
+      options:
+        - "unstable"
+        - "25.05"
+        - "24.11"
+        - "other"
+  - type: checkboxes
+    attributes:
+      label: I have read the FAQ
+      options:
+        - label: >-
+            I have read the [FAQ](https://nix-community.github.io/nixvim/user-guide/faq.html)
+            and my bug is not listed there.
+          required: true
+  - type: textarea
+    attributes:
+      label: Description
+      description: See https://stackoverflow.com/questions/29416909
```

Do these descriptions support markdown or html? It'd be nice to have a human readable link instead, e.g.:


```suggestion
      description: See [How can I write good bug reports?](https://stackoverflow.com/questions/29416909)
```